profile/README.md

```diff
@@ -69,6 +65,20 @@ interoperability and scalability compared to the previous generations.
 
 Virtualization is seen as a key innovation in order to fulfill these needs.
 
+## Start with SEAPAPTH
+
+SEAPATH comes with two main distributions, Yocto or Debian. You can find a comparaison of the two versions on the wiki [SEAPATH-Debian or SEAPATH-Yocto](https://lf-energy.atlassian.net/wiki/x/7I7lAQ)
```

Comment:
One typo and one suggestion:

[...] find a comparaison of the two versions on the wiki [...] ==> [...] find a **comparison** of **both** versions [...]